pkgs/development/python-modules/pygmt/default.nix

```
@@ -18,7 +18,7 @@
 
 buildPythonPackage rec {
   pname = "pygmt";
-  version = "0.9.0";
+  version = "0.10.0";
   format = "pyproject";
 
   disabled = pythonOlder "3.8";
```

```suggestion
  disabled = pythonOlder "3.9";
```
https://github.com/GenericMappingTools/pygmt/blob/e59b954dd34497a75f5c2386a4419910b44555db/pyproject.toml#L9